### PR TITLE
[WIP] Stop building and testing iOS simulator i368

### PR DESCRIPTION
**Pull request by shahmishal**

### CODE REVIEW

**Comment by shahmishal** [comment]
@swift-ci test

**Comment by shahmishal** [comment]
@swift-ci test macOS

**Comment by shahmishal** [comment]
@swift-ci test macOS

**Comment by shahmishal** [comment]
@swift-ci test macOS

File: cmake/modules/DarwinSDKs.cmake [hunk 1]
```diff
@@ -4,7 +4,7 @@ option(SWIFT_ENABLE_IOS32
 
 if(SWIFT_ENABLE_IOS32)
 set(SUPPORTED_IOS_ARCHS "armv7;armv7s;arm64;arm64e")
-set(SUPPORTED_IOS_SIMULATOR_ARCHS "i386;x86_64;arm64")
```
Comment (edymtt) [inline cmake/modules/DarwinSDKs.cmake:7]: (optional) This could be moved out of the if statement

Comment (shahmishal) [reply cmake/modules/DarwinSDKs.cmake:7]: I will update it, thanks!

**Pull request closed without merging**